drivers/msgbox/sunxi-msgbox.c

```diff
@@ -34,23 +34,23 @@
 #define XMIT_MSG_DATA_REG(n)    (0x0184 + 0x8 * (n))
 #define RECV_MSG_DATA_REG(n)    (0x0180 + 0x8 * (n))
 
-static inline msg_handler
+static inline msgbox_handler
```

Comment:
This header file is lacking documentation; adding it is a necessity for a refactoring PR.

Reply:
Documentation added.